### PR TITLE
[DOCS] 1.15 release notes

### PR DESCRIPTION
**Pull request by schavis**
🔍 [Deploy preview](https://vault-git-docs-release-notes-115-hashicorp.vercel.app/vault/docs/release-notes/1.15.0)

### CODE REVIEW

File: website/content/docs/release-notes/1.15.0.mdx [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+      &bull; Improved backend LDAP management and landing page dashboard.<br />
+      &bull; View secrets you have read access to under your directory.<br />
+      &bull; View diffs between previous and new secret versions.<br />
+      &bull; Copy and paste secret paths from the GUI to the Vault CLI or API.
+      <br /><br />
+      Learn more: <a href="/vault/docs/configuration/ui">Vault UI guide</a>
+    </td>
+  </tr>
+
+  <tr>
```
Comment (mladlow) [inline website/content/docs/release-notes/1.15.0.mdx:142]: Replication should probably go in the Enterprise section.

File: website/content/docs/release-notes/1.15.0.mdx [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>BETA</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Subscribe to notifications for various events in Vault. Includes support
+      for filtering, permissions, and cluster configurations with K-V secrets.
+      <br /><br />
+      Learn more: <a href="/vault/docs/concepts/events">Events</a>
+    </td>
+  </tr>
+
+  <tr>
```
Comment (finnstech) [inline website/content/docs/release-notes/1.15.0.mdx:109]: The audit logging work isn't customer facing, so I recommend removing this section.

File: website/content/docs/release-notes/1.15.0.mdx [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+    <td style={{verticalAlign: 'middle'}}>
+      New LDAP secrets engine GUI.
+      <br /><br />
+      Learn more: <a href="/vault/docs/configuration/ui">Vault UI guide</a>
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>ENHANCED</td>
+    <td style={{verticalAlign: 'middle'}}>
+      &bull; Improved backend LDAP management and landing page dashboard.<br />
```
Comment (finnstech) [inline website/content/docs/release-notes/1.15.0.mdx:133]: Suggest changing to New landing page dashboard, as LDAP is mentioned in previous section.
```suggestion
      &bull; New landing page dashboard.<br />
```

**Review by finnstech — CHANGES_REQUESTED** [review]

File: website/content/docs/release-notes/1.15.0.mdx (hunk 1 above)
Comment (schavis) [reply website/content/docs/release-notes/1.15.0.mdx:132]: Moved

File: website/content/docs/release-notes/1.15.0.mdx (hunk 2 above)
Comment (schavis) [reply website/content/docs/release-notes/1.15.0.mdx:109]: Removed

File: website/content/docs/release-notes/1.15.0.mdx [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+  </thead>
+  <tbody>
+  
+  <tr>
+    <td rowSpan={2} style={{verticalAlign: 'middle'}}>
+      Vault Agent
+    </td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>ENHANCED</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Updated to use the latest Azure SDK version and Windows Identity
```
Comment (finnstech) [inline website/content/docs/release-notes/1.15.0.mdx:67]: Replace `Windows Identity`

File: website/content/docs/release-notes/1.15.0.mdx [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+      Federation (WIF).
```
Comment (finnstech) [inline website/content/docs/release-notes/1.15.0.mdx:68]: Replace `Windows Identity Federation (WIF)` with `Workload Identity Federation (WIF)`

**Review by finnstech — CHANGES_REQUESTED** [review]

File: website/content/docs/release-notes/1.15.0.mdx (hunk 5 above)
Comment (schavis) [reply website/content/docs/release-notes/1.15.0.mdx:68]: fixed :)

**Review by mister-ken — APPROVED** [review]

**Review by jonathanfrappier — APPROVED** [review]

File: website/content/docs/release-notes/1.15.0.mdx [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      Seal High Availability
+    </td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>BETA</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Enables Vault administrators to configure multiple KMS for seal keys to
+      ensure Vault availability in the event a single KMS becomes unavailable.
+      <br /><br />
+      Learn more: <a href="#">TBD</a>
```
Comment (yhyakuna) [inline website/content/docs/release-notes/1.15.0.mdx:208]:
```suggestion
      Learn more: <a href="/vault/docs/configuration/seal/seal-ha">Seal High Availability</a>
```

**Review by finnstech — APPROVED** [review]

**Pull request merged**